src/olympia/devhub/cron.py

```diff
@@ -14,17 +14,27 @@
 
 def update_blog_posts():
     """Update the blog post cache."""
-    items = requests.get(settings.DEVELOPER_BLOG_URL, timeout=10).json()
-    if not items:
+    response = requests.get(settings.DEVELOPER_BLOG_URL, timeout=10)
+    items = response.json()
```

Comment:
Maybe handle the case where the response is not json at all ? It would throw here.

Reply:
it'd raise and the job would stop (without deleting the records) and get logged in sentry.  Is there something else we'd do though? Or it shouldn't raise?

Reply:
I guess it's fine to raise. Although maybe raising for the non-200 would make sense as well then